src/sentinel.c

```diff
@@ -3147,43 +3147,49 @@ void sentinelSendPeriodicCommands(sentinelRedisInstance *ri) {
 
 /* =========================== SENTINEL command ============================= */
 
-/* SENTINEL CONFIG SET <option> */
+/* SENTINEL CONFIG SET option value [option value ...] */
```

Comment:
There are few more places to have this command-usage fix. Please fix them as well.

Reply:
I have updated the the relevant parts. Let me know if anything is missed. Thanks